src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -12,7 +12,8 @@
     -->
 
     <!-- Publish trimmed single-file exe -->
```

Comment:
>trimmed [](start = 17, length = 7)

So is this comment now wrong?

Reply:
I've fixed. Were you really confused? If so, fine. 

I feel like the first comment explains the original intent, and the second comment (and commented setting) shows the change and documents why we're not using that setting. I left the setting in and commented to explain that we did consider PublishTrimmed but found problems with it. If I'd removed the PublishTrimmed line, we would lose the record that we tried it and had problems.

Overall, I feel like this combination of comments captures the initial intend and current state, and it's easy to understand which came first and which is current.

If you did understand the order, I feel like this feedback was overdoing it a little on comment pickiness. I don't want comments to be confusing or conflicting, but if people can understand the order of what happened and the comments collectively tell the right story, I feel like that's sufficient.